bazel_tools/scala.bzl

```diff
@@ -82,6 +83,7 @@ common_scalacopts = version_specific.get(scala_major_version, []) + [
     # Gives a warning for functions declared as returning Unit, but the body returns a value
     "-Ywarn-value-discard",
     "-Ywarn-unused:imports",
+    "-Ywarn-unused:-nowarn",
```

Comment:
What does this do?

Reply:
See the PR description. I'll add a comment here.